packages/manager/apps/web/client/app/domain/webhosting/order/domain-webhosting-order.routing.js

```diff
@@ -7,8 +7,17 @@ const resolve = {
   /* @ngInject */
   getAvailableModules: (cartId, WebHostingOrder) => (offer) =>
     WebHostingOrder.getAvailableModules(cartId, offer),
-  availableOffers: /* @ngInject */ (cartId, user, WebHostingOrder) =>
-    WebHostingOrder.getAvailableOffers(cartId, user.ovhSubsidiary),
+
+  /** TODO: for the moment the available offers must be hardcoded. Later they will be dynamic */
+  availableOffers: /* @ngInject */ () => [
```

Comment:
Maybe, you should move this part in a service even it's hard-coded for now. In this way, the routing part won't have to be modified when the offers will be "dynamic". Only the service logic will be change. What do you think ?

Reply:
why not, why i don't do it ?
currently the service already implement the available offers function.
But it return the no desired offers, so i guess once we have the required offers, we can reuse the updated endpoint.

Let me know

Reply:
No problem, it's ok for me